eng/Version.Details.xml

```diff
@@ -158,13 +158,13 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>cf64918877d98577363bb40d5eafac52beb80a79</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.ILAsm" Version="5.0.0-preview.4.20202.18">
+    <Dependency Name="Microsoft.NETCore.ILAsm" Version="5.0.0-preview.7.20316.4">
       <Uri>https://github.com/dotnet/runtime</Uri>
```

Comment:
If this is built out of this repo - why do we have a dependency on a pre-built version of it? Shouldn't we use what we just built?

Reply:
As discussed offline is because it is native and cross-compile complicates that:
https://github.com/dotnet/runtime/issues/1137